apps/backoffice-v2/src/lib/blocks/components/DirectorsCallToAction/interfaces.ts

```diff
@@ -0,0 +1,10 @@
+export interface ICallToActionDocumentOption {
+  name: string;
+  value: string;
+}
+
+export interface ICallToActionDocumentSelection {
+  options: ICallToActionDocumentOption[];
+  value?: ICallToActionDocumentOption['value'];
+  onSelect: (value: ICallToActionDocumentOption['value']) => void;
+}
```

Comment:
Previously this type was imported but it did not exist. We need to properly deprecate directors call to action.